packages/console/src/components/Tip/ToggleTip/index.module.scss

```diff
@@ -2,8 +2,8 @@
 
 .overlay {
   background: transparent;
-  position: fixed;
-  inset: 0;
+  padding: unset;
+  overflow-y: unset;
```

Comment:
I'm not sure, but is tooltip considered as an `overlay` now?

Reply:
The toggle tip uses react modal, and the tooltip uses createPortal. so this toggle tip should consider working correctly on react modal.